prow/kubeconfig.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
```

Comment:
Is this committed by mistake...? If yes, please be careful with files like this.

Reply:
these PRs are public btw

Reply:
Can load from l secret manager, only the token is sensitive

Reply:
Please remove it then. This file can be used to access the GKE cluster and cannot be public.